elevate-user/constants/routes.js

```diff
@@ -983,6 +983,17 @@ module.exports = {
 				functionName: 'userDetails',
 			},
 		},
+		{
```

Comment:
@MallanagoudaB all new configs we should add in utlis/interface-routes - qa-saas-routes.js